webpack.config.js

```diff
@@ -193,12 +193,12 @@ module.exports = (env, argv) => {
 
             // This fixes duplicate files showing up in chrome with sourcemaps enabled.
             // See https://github.com/webpack/webpack/issues/7128 for more info.
-            namedModules: false,
+            moduleIds: "named",
```

Comment:
This seems to have changed which implies the comment is no longer valid

Reply:
Oh, actually I think I got it the wrong way around. The migration guide had

```
optimization.namedModules: true → optimization.moduleIds: 'named'
```

but our usage was with `false`. 🤦

Reply:
The [original issue](https://github.com/webpack/webpack/issues/7128) was closed due to inactivity in 2019 which makes me suspect that this isn't actually an issue anymore.

Reply:
Happy if the solution is to remove the comment

Reply:
Comment removed